lib/models/ObjectLockConfiguration.js

```diff
@@ -199,6 +199,38 @@ class ObjectLockConfiguration {
         }
     }
 
+    /**
+     * Get the XML representation of the configuration object
+     * @param {object} config - The bucket object lock configuration
+     * @return {string} - The XML representation of the configuration
+     */
+    static getConfigXML(config) {
```

Comment:
Could be good to have some basic tests for this function

Reply:
I just saw there are some in cloudserver. Could it make more sense to have the unit tests moved to arsenal? At least those that do not have a dependency on cloudserver.

Reply:
Since it's a model class, it should be easy to test part of it as unit tests, the functional tests that use the S3 API make sense too but it's good to have some local unit tests as well.

Reply:
Yeah, like you mentioned it’s indirectly getting unit tested in cloudserver, also I realized we don’t have unit tests for any of the similar static class methods under other configuration models.
I still wanted to add some unit tests instead of going along with the current pattern though, yet, it hasn’t been as easy as I imagined due to the limitations of testing a static class method by calling it on the instance of the class (which I need to be doing for the unit test). 

As an alternative, I made the method non-static to be able to call it on the instance of the class (as static method calls are made directly on the class and are not callable on instances of the class) and to be able to unit test it, but suggestions for a better approach are welcome.

Reply:
You should be able to test static methods by calling them like `ObjectLockConfiguration.getConfigXML(...)`, I think? Or maybe I'm missing your point.

Reply:
I think it should work by calling it on the class, but for the unit test, I needed to call it on the instance of the class which is when it becomes tricky. But I don't necessarily have to call it on the instance in the first place, in fact, for the unit test, probably it should suffice to just unit-test the method alone by calling it on the class. I'll do that.

Reply:
Yes that's what I mean, you can test this method separately as a class method test, and have other instance method tests.

Reply:
@jonathan-gramain tests added, can you take another look ?